spec/controllers/queue_items_controller_spec.rb

```diff
@@ -119,20 +119,20 @@
       end
 
       it "redirects to the my queue page if authenticated" do
-        post :update_queue,
-             queue_items_data: {:list_orders => [], :ratings => []}
+        post :update_queue, params:
+          { queue_items_data: { list_orders: [5], ratings: [3] } }
         expect(response).to redirect_to queue_items_path
       end
 
       it "reorders the queue items" do
-        post :update_queue,
-             queue_items_data: { :list_orders => [3, 2, 1], :ratings => [] }
+        post :update_queue, params:
+          { queue_items_data: { list_orders: [3, 2, 1], ratings: [nil] } }
```

Comment:
I don't think `nil` is a value that can ever be in params, I think it would be empty string coming from the browser (or an empty array as you used to have). It is important to test using real life data so you don't have false positives/failures.